lib/membrane_rtc_engine/endpoints/rtsp/connection_manager.ex

```diff
@@ -0,0 +1,286 @@
+if Enum.all?(
+     [
+       Connection,
```

Comment:
We lack UDP.Source. Let's define `Endpoint.RTSP.get_required_deps` and use it here instead

Reply:
Yeah I know but the idea is to require the same set of deps in every module related to RTSP